src/attrs.rs

```diff
@@ -64,7 +64,7 @@ fn is_relevant_trait(item: &TraitItem) -> bool {
 }
 
 fn is_relevant_block(block: &Block) -> bool {
-    for stmt in block.stmts.iter() {
+    for stmt in &block.stmts {
```

Comment:
Suggestion: Let's add `cargo rustc -- -L target/clippy.so -Z extra-plugins=clippy` to our travis build.